### DISCUSSION TITLE
Feature request - handle Bezier handles seperately

### DISCUSSION THREAD

**Discussion by shimonshomron** [keiwando/toonsquid#68] [Ideas]
Currently - when creating/editing vector shape with Bezier handles I can't control each side of the handle separately - which make the curve to be affected on both sides of the point. 
On every vector editing app/software - you can manipulate each side's handle individually and by this accomplish any type of curve you like. 
(In general with touch interface like an iPad commonly use is holding the flinger down while moving the Bezier handle with the pencil makes it move solely without affecting the other side handle)

**Comment by keiwando** [comment]
Hi and thanks for all of your suggestions!
You can currently tap a path control point and change its mode to define how the handles at that point should behave. If you drag out handles while initially placing the control point, the mode defaults to `Symmetric` but you can also set it to `Asymmetric` or `Disconnected`:

https://toonsquid.com/handbook/path/path_tool/#control-point-mode

**Reply by shimonshomron** [reply]
Well first, That's great to know that it's possible, when played with it I was sure its there and I'm missing it.
The next suggestion is to be able to be more fluent with that - because complex vector shapes can be pain in the *** :) generally speaking. so a more fluent approach which keep you running with the line instead of opening afterwards each point can make the process much smoother and quicker.
One hand with the pencil moving the handle, while if you touch/hold on screen with the other hand's thumb - breaks the handle to be "disconnected". This is how other vector apps approach it (which makes it also more familiar for newbies with your fine app like me :)

**Reply by keiwando** [reply]
Yes, absolutely! I've already noted this down as a potential improvement for the path editing experience. Please keep bringing up points of frustration like this, no matter how small. This is very helpful!
